web/views/index.php

```diff
@@ -4,19 +4,19 @@
 
 // Default parameters if not specified
 $defaults = [
-    "color" => "brightgreen",
+    "color" => "blue",
```

Comment:
Default color for views should be blue. Let me know when that's changed and I think it's ready to merge. 🎉

Reply:
It is blue in this latest update, or you meant any other colour.?

Reply:
Okay, I checked it would be brightgreen, I will change it now..

Reply:
Sorry, I meant "brightgreen" like before and not blue.

Reply:
I changed the view index.php default color to bright green, respectively. Please review it...

Thank you again for all your advice.